package.json

```diff
@@ -43,33 +43,35 @@
     "npm": ">=3.0.0"
   },
   "devDependencies": {
-    "aegir": "^21.4.5",
+    "aegir": "^21.8.1",
     "chai": "^4.2.0",
     "dirty-chai": "^2.0.1",
-    "lodash": "^4.17.11",
+    "just-range": "^2.1.0",
     "memdown": "^5.1.0",
     "multihashes": "~0.4.15",
     "multihashing-async": "~0.8.0",
     "ncp": "^2.0.0",
     "rimraf": "^3.0.0",
-    "sinon": "^9.0.1"
+    "sinon": "^9.0.2"
   },
   "dependencies": {
-    "base32.js": "~0.1.0",
     "bignumber.js": "^9.0.0",
+    "buffer": "^5.5.0",
     "bytes": "^3.1.0",
-    "cids": "^0.8.0",
-    "datastore-core": "~0.7.0",
+    "cids": "~0.8.0",
```

Comment:
Why has this changed from `^` to `~`?

Reply:
i don't know probably happened in the rebase, but ~ is what we use for below  1.0.0 right ?

Reply:
We used to have a linting rule that enforced that, then we relaxed it because npm uses `^` for above 0.0.x

Reply:
i have changed it back